src/main/java/co/rsk/federate/signing/SignerVersion.java

```diff
@@ -0,0 +1,20 @@
+package co.rsk.federate.signing;
+
+public enum SignerVersion {
```

Comment:
Version 1 is a key file. This is why I named it as `SignerVersion`.

Reply:
I understand your point better now, and I kind of agree with you, but that refactor is more extensive than you may think. Check [this class](https://github.com/rsksmart/powpeg-node/blob/77c22f1d9c2458e53a2606e60677432ec1b6f11f/src/main/java/co/rsk/federate/signing/hsm/client/HSMSigningClientProvider.java#L47) out please. There is an `HSMSigningClientV1`, which, from my understanding, should not be named with the prefix `HSM.` since I understand `HSM` is not for `v1`.

So, I think a couple of refactors/improvement opportunities should be addressed in its specific task, but not this one because it looks out of scope to me.

Let me know what you think. Thanks.

Reply:
Done